### PR TITLE
 [ENDPOINT] Add Pagination to News OT246-86

### PR DESCRIPTION
**Pull request by Samuel0023**
## Description
AS user
I WANT to view the results of Paginated News
TO increase response speed

Criteria of acceptance:

When the request to list is made, the results should be paginated by 10. In the response, the results and the urls for the previous and next page (if applicable) should be shown. The current page will be obtained from the query parameter ?page

## Evidence 
example: 13 news
- page 1

image
- page 2

image

### CODE REVIEW

File: config/config.js [hunk 1]
```diff
@@ -9,6 +9,7 @@ module.exports = {
     host: process.env.DB_HOST,
     port: process.env.DB_PORT,
     dialect: 'mysql',
+    baseURL: process.env.BASE_URL,
```
Comment (ghnoob) [inline config/config.js]: este archivo no es para la confi de sequelize? o se puede usar para todo?

Comment (Samuel0023) [reply config/config.js]: Y he visto ejemplos, que lo ponían allí. Pero ahora que me acuerdo, no lo uso

Comment (Samuel0023) [reply config/config.js]: xD

**Review by ghnoob — APPROVED** [review]
Muy bueno el helper que creaste

**Review by ignacioborraz — APPROVED** [review]
ME
EN
CA
NT
O!
jajajaja

File: helpers/paginator.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    const baseURL = process.env.BASE_URL
+    return { prevUrl: this.prevUrl(baseURL), nextUrl: this.nextUrl(baseURL) }
+  }
+
+  getParams() {
+    return { offset: this.getOffset(), limit: this.getLimit() }
+  }
+}
+
+module.exports = Paginator
```
Comment (gabodmc) [inline helpers/paginator.js:48]: Niiiiiiiiiiiiiiiiiiiiiiice!

**Review by gabodmc — APPROVED** [review]

**Pull request merged**